helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -49,7 +49,9 @@ receivers:
   {{- if (eq (include "splunk-otel-collector.metricsEnabled" .) "true") }}
   hostmetrics:
     collection_interval: 10s
-    root_path: {{ .Values.isWindows | ternary "C:\\hostfs" "/hostfs" }}
+    {{- if not .Values.isWindows }}
```

Comment:
This will work assumes that all images are either Linux or Windows - I *think* that is true. The ideal fix would be `if .Values.isLinux` but I don't know if that is a possibility.

Reply:
yes so far only linux and windows are supported